docs/fides/docs/guides/generate_resources.md

```diff
@@ -22,6 +22,7 @@ A connection string can be supplied using the `connection-string` option:
 ...
 ```
 
+The appropriate `connection-string` format for your database connector can be found in the [Sqlalchemy Documentation](https://docs.sqlalchemy.org/en/14/dialects/mysql.html#dialect-mysql-aiomysql-connect).
```

Comment:
```suggestion
The appropriate `connection-string` format for your database connector can be found in the [SQLAlchemy Documentation](https://docs.sqlalchemy.org/en/14/dialects/).
```
I wasn't sure on if there was a reason the specific connection/dialect was chosen, so thought this link might work as a landing spot for different dialects